Doc/glossary.rst

```diff
@@ -758,6 +746,18 @@ Glossary
       :term:`finder`. See :pep:`302` for details and
       :class:`importlib.abc.Loader` for an :term:`abstract base class`.
 
+   locale encoding
+      On Unix, it is the encoding of the LC_CTYPE locale. It can be set with
+      :func:`locale.setlocale(locale.LC_CTYPE, new_locale) <locale.setlocale>`.
+
+      On Windows, it is the ANSI code page (ex: ``"cp1252"``).
+
+      On Android and VxWorks, Python uses ``"utf-8"`` as the locale encoding.
+
+      ``locale.getencoding()`` can be used to get the locale encoding.
```

Comment:
Maybe a stretch here, but while we're touching these lines anyway, we could fix the incorrect formatting here and make this a proper cross-reference, so readers can easily jump directly to the relevant function doc instead of having to try to find and navigate to it manually.

```suggestion
      :func:`locale.getencoding` can be used to get the locale encoding.
```

Reply:
Good suggestion @CAM-Gerlach. I only looked at the alpha sorting.